staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -95,7 +95,9 @@ func DefaultStacktracePred(status int) bool {
 	return (status < http.StatusOK || status >= http.StatusInternalServerError) && status != http.StatusSwitchingProtocols
 }
 
-const withLoggingLevel = 3
+// default logging level INFO is 4. Let's make sure if folks really need the http logs they
```

Comment:
I don't think default log level is 4, I think it's 2. It gets bumped up in test, right?

Reply:
you are right. fixed in latest PR